docs/pages/application-access/cloud-apis/azure.mdx

```diff
@@ -30,6 +26,15 @@ prevent unauthorized access to your organization's Azure identities.
 
 - An Azure VM where you will run the Teleport Application Service. The Azure VM
   must be running a Linux distribution.
+  
+  <Admonition type="note" title="Azure Kubernetes Service (AKS)">
+  Note that this guide is also applicable to Azure Kubernetes Service (AKS)
+  deployment with Microsoft Entra pod-managed identities enabled. However, the
+  pod-managed identities feature will deprecate in September 2024.
```

Comment:
Nit: I don't remember seeing `deprecate` used in this fashion... perhaps we can rewrite this way?

```suggestion
  pod-managed identities feature will be deprecated in September 2024.
```